webanno-io-tsv/src/main/java/de/tudarmstadt/ukp/clarin/webanno/tsv/WebannoTsv1Reader.java

```diff
@@ -101,7 +101,7 @@ private void createToken(JCas aJCas, StringBuilder text, Map<Integer, String> to
             Map<Integer, String> pos, Map<Integer, String> lemma, Map<String, Token> tokensStored)
     {
         int tokenBeginPosition = 0;
-        int tokenEndPosition = 0;
+        int tokenEndPosition;
```

Comment:
I prefer having explicit initalizers for primitive variables, even if they initialize to the same value that the JVM would use.